code/modules/mob/living/carbon/carbon.dm

```diff
@@ -200,7 +200,12 @@
 	if(istype(potential_spine))
 		extra_throw_range += potential_spine.added_throw_range
 
-	newtonian_move(get_dir(target, src))
+	var/drift_force = max(0.5, 1 + power_throw)
```

Comment:
can power_throw be negative? what?

Reply:
also where is this coming from (what units?)

Reply:
in case of it being negative we get 0.5 newtons of drift force